setup.py

```diff
@@ -53,7 +53,7 @@ def read(fname):
                       'pandas', 'Pillow', 'h5py'],
     extras_require={
         'test': ['tox', 'flake8', 'pytest', 'pytest-cov', 'pytest-xdist', 'mock',
-                 'sagemaker>=1.15.2', 'tensorflow', 'docker-compose'],
+                 'sagemaker>=1.19.1', 'tensorflow', 'docker-compose'],
```

Comment:
let's pin this to reduce the blast radius of potential issues with Python SDK releases